docs/projects/version-control/config-as-code-reference.md

```diff
@@ -134,6 +127,30 @@ If multiple projects will be persisted to the repository, adding the project nam
 
 We recommend storing projects alongside the application code. While it is possible to store all your deployment projects in a single central repository with folders for each project, it will be challenging to manage as you add more projects. For example, if you have multiple component projects, one for Web UI, another for Web API, etc., but the source code is in one repository, then store all the component projects in that repository. If you move the application code later, you can also [move the deployment configuration](/docs/projects/version-control/moving-version-control.md) to keep it with the application.
 
+### Branch Settings
+
+#### Default Branch Name
+
+The _Default Branch Name_ is the branch on which the Octopus configuration will be written. It is also the default branch that will be used in various situations, for example:
+
+- When users view the project's deployment process for the first time in the Octopus UI, this is the initially selected branch 
+- When creating releases, this will be the default branch selected
+- When running Runbooks, variable values will be pulled from this branch
+
+For existing initialized repositories, the default branch must exist. If the repository is new and uninitialized, Octopus will create the default branch automatically.
+
+#### Initial Commit Branch
+
+If the default branch is protected in your repository, select the *Is the default branch protected?* checkbox. This will give you the option to use a different _Initial Commit Branch_. If this branch does not existing, Octopus will create the branch automatically. 
```

Comment:
```suggestion
If the default branch is protected in your repository, select the *Is the default branch protected?* checkbox. This will allow you to use a different _Initial Commit Branch_. If this branch does not exist, Octopus will create the branch automatically. 
```